tools/dockerfile/interoptest/grpc_interop_java_oracle8/build_interop.sh

```diff
@@ -16,16 +16,24 @@
 # Builds Java interop server and client in a base image.
 set -e
 
-mkdir -p /var/local/git
-git clone --recursive --depth 1 /var/local/jenkins/grpc-java /var/local/git/grpc-java
+cp -r /var/local/jenkins/grpc-java /tmp/grpc-java
```

Comment:
Hmm, IMHO `git clone` is the the safest option over `cp`, since with `git clone`, it's easier to be sure that the resulting repo in the container is at a mint state.

Reply:
During "real" builds, it will be a fresh checkout. In dev builds, it is bizarre to me to take the latest commit, which is likely a local-only dev commit, and ignore any changes in progress.

I do agree it would be nice to ignore any build outputs already generated. Sort of like specifying $outdir to configure/make. I did toy with it a little for gradle, but it appears to be a very incomplete feature. In any case, gradle does a great job checking incremental builds, so there's little worry of it doing the wrong thing and it would only "fail" for dev builds, which wouldn't be unexpected.